southworkscom/AI
# [Enterprise Bot][TypeScript] Update Middlewares

## Description

The structure of the middlewares between C# and TypeScript contains a similar information. For this purpose we changed the following things:
- Change the project structure on Typescript. 
- Add telemetryBotDependencyExtension
- The file SetLocaleMiddleware is not necessary on Typescript version
- Update the files inside the telemetry folder
- Update the files index.ts and mainDialog.ts to use the new middlewares


## Related Issue

Fixes #561

## Testing Steps

1. Open `AI/templates/Enterprise-Template/src/typescript/src/middleware/telemetry` and 
`AI/templates/Enterprise-Template/src/csharp/EnterpriseBotSample/Middleware/Telemetry`.
2. Check the structure between them.

## Checklist
~- [] I have added or updated the appropriate unit tests~
~- [] I have updated related documentation~

If this contains changes that needs to be replicated between the Enterprise Template <-> Virtual Assistant
~- [ ] A duplicate issue is filed to track future  work.~

If you have updated responses or `.lu` files:
~- [] All languages have been updated~
